src/pages/signin/SignInPageLayout/Footer.js

```diff
@@ -13,16 +14,37 @@ import Licenses from '../Licenses';
 import Socials from '../Socials';
 import Hoverable from '../../../components/Hoverable';
 import CONST from '../../../CONST';
-import Navigation from '../../../libs/Navigation/Navigation';
+import Navigation, {navigationRef} from '../../../libs/Navigation/Navigation';
 import * as Session from '../../../libs/actions/Session';
+import screens from '../../../SCREENS';
 
 const propTypes = {
+    scrollViewRef: PropTypes.shape({
+        // eslint-disable-next-line react/forbid-prop-types
+        current: PropTypes.any,
+    }),
     ...windowDimensionsPropTypes,
     ...withLocalizePropTypes,
 };
 
-const navigateHome = () => {
-    Navigation.navigate();
+const defaultProps = {
+    scrollViewRef: {current: null},
```

Comment:
i believe this is more accurate

```suggestion
    scrollViewRef: undefined,
```